default.env

```diff
@@ -11,3 +11,22 @@ HC_IMAGE_TAG=
 L2GETH_HTTP_PORT=9991
 L2GETH_WS_PORT=9992
 DTL_PORT=7878
+
+# Secure web proxy - advanced use, please see instructions at https://eth-docker.net/docs/Usage/ReverseProxy
```

Comment:
All of this should go into another env file.
Multiple env files can included like https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option

Reply:
It needs to be in `.env` for docker-compose to have it for the yml. `env-file` just brings it into the container. I've moved this to a contrib `default.env` with instructions to use it when using traefik.